app-i18n/uchardet/uchardet-9999.ebuild

```diff
@@ -7,10 +7,7 @@ inherit cmake-utils git-r3
 
 DESCRIPTION="An encoding detector library"
 HOMEPAGE="https://www.freedesktop.org/wiki/Software/uchardet/"
-EGIT_REPO_URI=(
-	https://anongit.freedesktop.org/git/${PN}/${PN}.git
-	git://anongit.freedesktop.org/${PN}/${PN}
-)
+EGIT_REPO_URI="https://anongit.freedesktop.org/git/${PN}/${PN}.git"
```

Comment:
> HTTPS blocking is actually the reason why unsafe fallback options are a bad idea for the majority of users who aren't subject to HTTPS blocking.

Sigh, the majority again. What about the rest, who would like to use Gentoo in places where they don't have control over network setup? Constantly flip ${PN}_LIVE_REPO?
I agree, the current situation is not ideal from the security point of view, but provides zero PITA for users.

> Security matters aside, I'd also be surprised if port 443 was blocked more frequently than 9418 (git transport). For this reason, HTTPS is probably the most likely protocol of the two to be allowed through on the first attempt.

uchardet already probes HTTPS first.

> Some solutions to support your arguments, as well as the secure protocol only argument are:
...

All of these can be done by users who afraid that somebody will read their git traffic and want HTTPS.